### PR TITLE
Added Popular menu

### PR DESCRIPTION
**Pull request by chococoul**

### CODE REVIEW

File: src/frontend/components/PopularMenu.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
+
+export default class PopularMenu extends Component {
+    render() {
```
Comment (aduwillie) [inline src/frontend/components/PopularMenu.js]: This rendering is just too big. Consider breaking it down into simple components.

File: src/frontend/components/style.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+$blinkas-red: #ff0000;
```
Comment (aduwillie) [inline src/frontend/components/style.scss]: This is just a repetition of styles you have created in another PR. If there happen to be dependencies in your work, kindly prompt your instructor for help on it.

File: src/frontend/components/test-utils.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (aduwillie) [inline src/frontend/components/test-utils.js]: this space is not needed.

File: src/frontend/components/PopularMenu.js (hunk 1 above)
Comment (chococoul) [reply src/frontend/components/PopularMenu.js]: Do you want me to break individual `list` into components?

**Pull request merged**